src/bsl/tree-sitter/codeModelFactory.ts

```
@@ -0,0 +1,366 @@
+    buildModel(parser: BslParser) {
+        const start = performance.now()
+        const model = new BslCodeModel()
+        const root = parser.getRootNode()
+        fillChildren(root.children, model.children)
+        console.debug('Build code model', performance.now() - start, 'ms')
+        return model
+    },
+}
+
+export function createSymbol(node: Node): BaseSymbol | BaseSymbol[] | undefined {
+    const constructor = constructors[node.type]
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)